DigitalLearningSolutions.Web/Views/LearningMenu/Index.cshtml

```diff
@@ -29,8 +31,23 @@
 </div>
 
 <div>
-  @foreach(var section in Model.Sections)
+  @foreach (var section in Model.Sections)
   {
     <partial name="_SectionCard" model="section" />
   }
 </div>
+
+<div class="nhsuk-card incomplete">
+  <div class="nhsuk-card__content">
+    <h2 class="nhsuk-card__heading nhsuk-heading-m">
+      Completion Status: Incomplete

```

Comment:
Might be worth changing to "Course Completion Status: ..." just to be completely clear? 🙂